### ISSUE TITLE
Add all contributors of a PR

### ISSUE THREAD

**Issue by Steellgold** [release-drafter/release-drafter#1333]
In fact, only the author of the PR is mentioned in the release, but if there are contributors to the press release, they are not mentioned in the release.

**Comment by Steellgold** [comment]
Exemple PR by @Bluzzi: 
![image](https://user-images.githubusercontent.com/51505384/230048520-6f56a79f-5162-43e6-8de8-8f87366a01eb.png)

But it was I who made the updates in the PR
![image](https://user-images.githubusercontent.com/51505384/230048780-b9aad078-b13e-4291-88ce-5ef2903f1e47.png)
![image](https://user-images.githubusercontent.com/51505384/230048806-914acd47-f9ab-4c7f-9bec-68d0c953f97f.png)

And in the release:
![image](https://user-images.githubusercontent.com/51505384/230049033-4ad01299-79d6-48b8-94ec-542e1a41aa6a.png)

**Comment by jetersen** [comment]
Yes it would be nice to include `Co-authored-by` commit trailers however it seems that humans like to remove the `Co-authored-by` even though they contributed. See this as an example: https://github.com/jenkinsci/jenkins/commit/4e9cc522bbab6ef69307743d0bafb28cbab398cc

Could you perhaps instead of a screenshot share the `.patch` like this: https://github.com/jenkinsci/jenkins/commit/4e9cc522bbab6ef69307743d0bafb28cbab398cc.patch

**Comment by ssbarnea** [comment]
This is really an important fix to add to release drafter because we face other the need to add final fixes to a PR that originated from an outside contributor and if we cannot push to his own fork (quite common), we are forced to create a branch and make a PR ourselves. While we do keep the original commit author, the PR author changes to one of us (maintainers), and usually we want to send the credits/kudos to the external contributor.

**Comment by cchanche** [comment]
Hm I see, so there are two different things here

- PRs that were worked-on by multiple people
- Commits that were authored by multiple people at once (using the `Co-authored-by` header)

Current implementation only supports mentioning the author or the PR in the release.

To remediate, there would be two major difficulties :
1. list every other contributor
2. find a nice and generic way to provide their names in templatable variables

Step 1. may require a whole lot of additional fetching, but nothing too hard I guess. We would have to pay attention to every possible type of merging strategies (merge, rebase & squash), such as we don't loose contribution metadata along the way.

Step 2. might be a bit more tricky to nail, but nothing's impossible. What if there's more than 2 contributors ? More than 3 ? What if 99% of the commits were added by one person, and others did almost nothing ? Do we perform some sort of sorting ? By # of commits ? By # of changes ?

Contributions are welcome !
